UITests/BraintreeDropIn_UITests.swift

```diff
@@ -552,45 +552,56 @@ class BraintreeDropIn_PayPal_UITests: XCTestCase {
         waitForElementToBeHittable(app.staticTexts["Credit or Debit Card"])
         XCTAssertTrue(app.staticTexts["PayPal"].exists)
     }
-    
+
     func testDropIn_paypal_receivesNonce() {
-        if #available(iOS 11.0, *) {
-            // SFSafariAuthenticationSession flow cannot be fully automated, so returning early
-            return
-        }
-        
+        // Tap Drop-in PayPal button
         waitForElementToBeHittable(app.staticTexts["PayPal"])
         app.staticTexts["PayPal"].tap()
         sleep(3)
-        
+
+        // Tap "Continue" on alert
+        addUIInterruptionMonitor(withDescription: "Alert prompting user that the app wants to use PayPal.com to sign in.") { (alert) -> Bool in
+            let continueButton = alert.buttons["Continue"]
+            if (alert.buttons["Continue"].exists) {
+                continueButton.tap()
+            }
+            return true
+        }
+        app.tap()
+
+        // Interact with PayPal webview
         let webviewElementsQuery = app.webViews.element.otherElements
-        
         waitForElementToBeHittable(webviewElementsQuery.links["Proceed with Sandbox Purchase"])
-        
         webviewElementsQuery.links["Proceed with Sandbox Purchase"].forceTapElement()
-        
+
+        // Assert on demo
         waitForElementToAppear(app.staticTexts["bt_buyer_us@paypal.com"])
-        
         XCTAssertTrue(app.staticTexts["bt_buyer_us@paypal.com"].exists)
     }
```

Comment:
**Note:** previously these PayPal UI test weren't running at all when running on an iOS 11+ simulator. I took a stab at making them work. I got them working, though they are a bit buggy - they only work on simulators running iOS 13 or 14, not below for some reason.

Do we want to keep these tests and write a comment, or nix them entirely? What about adding an integration test for a PayPal success case instead?

Reply:
I'd be OK with keeping them. We'll probably be testing on iOS 13 and 14 for the most part anyway.

An integration test is another good idea, although I'm not sure exactly what that would look like, since the Drop-in code is so UI-heavy.

Reply:
> An integration test is another good idea, although I'm not sure exactly what that would look like, since the Drop-in code is so UI-heavy.

True - I think it would be boil down to calling the `BTPayPalDriver` directly, which is essentially the[ PayPal IntegrationTests in `braintree-ios`](https://github.braintreeps.com/team-sdk/braintree-ios/blob/master/IntegrationTests/BraintreePayPal_IntegrationTests.m).

Reply:
Hey @sestevens - I left these tests in and added a comment to their flakiness. I opted to not use the `if #available(iOS 13.0, *)` annotation since I think having them fail is better than silently not execute. Let me know what you think 👍

Reply:
Makes sense to me!